react/src/lib/Input/tests/index.spec.js

```diff
@@ -17,6 +17,14 @@ describe('tests for <Input />', () => {
     expect(container).toMatchSnapshot();
   });
 
+  it('should match multiline SnapShot', () => {
```

Comment:
Better description here since the test was changed in that last commit - should change tag based on multiline prop.

Reply:
I've updated the description. Still need to squash the commits